doc/tutorial/code/solutions/Ex12b1.Format.fst

```diff
@@ -80,21 +84,23 @@ let response s t =
    functions---they just return messages---so these three lemmas are
    sufficient *)
 
+// BEGIN: FormatLemmasEx
 val req_resp_distinct:
   s:string -> s':string16 -> t':string ->
   Lemma (requires True)
-        (ensures (request s <> response s' t'))
+        (ensures  True) //replace with actual lemma post-condition
```

Comment:
Why you didn't do it?

Reply:
Interesting, I don't remember doing this. May be an unintentional change that I copied over from the exercises (vs solutions).